examples/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+set(pkgName "kubernetes")
```

Comment:
The `pkgName` should not be "kubernetes", I think "kubernetes_examples" is better ("kubernetes" is the name of library), but I agree this code can deliver now, we can update `CMakeLists.txt` in future.

Reply:
pkgName variable is used in the examples to find the installed Kubernetes client package. If we change its name here the installed package will not be found.

Reply:
I mean setting "pkgName=kubernetes_examples", and link Kubernetes library such as
``` 
        include_directories(${KUBER_INCLUDE_DIR})
        link_directories(${KUBER_LIBRARIES})
```

We can deliver your code now to focus on the main function.

Reply:
That's not the correct way to do it. Examples should reflect correct CMake usage and correct usage should be done using find_package() and imported components.

Reply:
My comment is based on the suggestion that "examples" is a sperate project that links the other sperate project "kubernetes library".
Let me file a PR some times later and ask your review~